vidify/gui/components.py

```diff
@@ -6,12 +6,12 @@
 import logging
 from typing import Optional
 
-from PySide2.QtWidgets import (QWidget, QLabel, QPushButton, QLineEdit,
+from qtpy.QtWidgets import (QWidget, QLabel, QPushButton, QLineEdit,
                                QVBoxLayout, QGridLayout, QGroupBox)
```

Comment:
./vidify/gui/components.py:10:32: E127 continuation line over-indented for visual indent

you need to remove a couple spaces here too